dataland-frontend/tests/e2e/fixtures/fakeFixtures.ts

```diff
@@ -120,6 +122,7 @@ function generateCSVData(companyInformationWithEuTaxonomyData: Array<Object>) {
             {label: 'Unternehmensname', value: 'companyName'},
```

Comment:
Ich finde einige Benamungen in diesem File zu generisch und nichtssagend.
Ich würde hier einiges umbenennen.

Reply:
Wurde von Siegfried verbessert